site/content/blogs/release-v118.en.md

```diff
@@ -22,8 +22,69 @@ enabling you to focus on developing your application, instead of writing deploym
 See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## Certificate import
+
 _Contributed by [Penghao He](https://github.com/iamhopaul123/)_
 
+Developers that have domains managed outside of Route 53 or who want to use HTTPS for Copilot applications without any domain specified can now use a new `--import-cert-arns` flag to import any validated certificates when they create an environment.
```

Comment:
nit: maybe this will break down the sentence a little bit? 

```suggestion
If you have domains managed outside of Route 53, or want to enable HTTPS without having domain associated with your application, you can now use the new `--import-cert-arns` flag to import any validated certificates when creating your environments.
```